cmd/api/src/model/queries.go

```diff
@@ -0,0 +1,66 @@
+package model
+
+import "fmt"
+
+type SavedQuery struct {
+	UserID string `json:"user_id"`
+	Name   string `json:"name"`
```

Comment:
where do we typically define sql-level constraints for a model, such as the fact that there is a uniqueness constraint on the `name` field?